### PR TITLE
Update sonata-project/datagrid-bundle version

### PR DESCRIPTION
**Pull request by c-Rolland**
## Subject

I am targeting this branch, to update the version of **sonata-project/datagrid-bundle** form **2.3** to **3.2**

## Changelog
```markdown
### Changed
- Constraint for `sonata-project/datagrid-bundle` dependency
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "sonata-project/admin-bundle": "^3.58",
         "sonata-project/block-bundle": "^3.18",
         "sonata-project/cache-bundle": "^2.4.2",
-        "sonata-project/datagrid-bundle": "^2.3",
+        "sonata-project/datagrid-bundle": "^3.2",
```
Comment (phansys) [inline composer.json:31]: It's not just adding support, but bumping the major constraint.

**Review by phansys — COMMENTED** [review]
Thanks for the contribution.
Could you please fill the PR template in order to make clear the changes made here?

File: UPGRADE-0.x.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 UPGRADE 0.x
 ===========
 
+### Add support for sonata-project/datagrid-bundle:^3.2
```
Comment (phansys) [inline UPGRADE-0.x.md:4]: Bumping a major constraint is not a synonym of adding support.
Adding support is the case when you add support over the existing compatibility, IE `^2.3|^3.2`. But you are dropping support for `^2.3` here.

Comment (c-Rolland) [reply UPGRADE-0.x.md:4]: ahh sry, i forget to update it

File: UPGRADE-0.x.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 UPGRADE 0.x
 ===========
 
+### Update SonataDatagridBundle version
+Update **sonata-project/datagrid-bundle** version (from 2.3 to 3.2)
```
Comment (phansys) [inline UPGRADE-0.x.md:5]:
```suggestion

The constraint for `sonata-project/datagrid-bundle` was bumped to `^3.2`.
```

File: UPGRADE-0.x.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 UPGRADE 0.x
 ===========
 
+### Update SonataDatagridBundle version
```
Comment (phansys) [inline UPGRADE-0.x.md:4]:
```suggestion
### Dependencies
```

**Review by phansys — CHANGES_REQUESTED** [review]

File: UPGRADE-0.x.md (hunk 2 above)
Comment (phansys) [reply UPGRADE-0.x.md]: Almost there.

**Comment by phansys** [comment]
Please, update the **Changelog** section in order to respect the PR template:
```markdown
### Changed
- Constraint for `sonata-project/datagrid-bundle` dependency
```

**Comment by c-Rolland** [comment]
@phansys hello ^^ little question : when is it the next major release ?  **sonata-project/datagrid-bundle** lock the **sonata-project/datagrid-bundle** version and causes conflicts between others sonata dependencies. I cannot update the version of  **sonata-project/notification-bundle** for exemple ...

**Review by VincentLanglet — DISMISSED** [review]

**Review by wbloszyk — CHANGES_REQUESTED** [review]
Look at:
https://github.com/sonata-project/SonataUserBundle/pull/1230/files

https://github.com/sonata-project/SonataDashboardBundle/blob/89266fc82504e71fa11875c91dcfda7eaf064e78/src/Entity/BlockManager.php#L77

https://github.com/sonata-project/SonataDashboardBundle/blob/89266fc82504e71fa11875c91dcfda7eaf064e78/src/Entity/DashboardManager.php#L28

You have to update these implementation too.

**Comment by VincentLanglet** [comment]
Should we add test to detect this ?

**Review by wbloszyk — APPROVED** [review]

**Pull request closed without merging**